guides/payments.md

```diff
@@ -0,0 +1,220 @@
+# Payments
+
+## Overview
+
+Solidus has a highly flexible payments model which allows multiple payment
+methods to be available during checkout. The logic for processing payments is
+decoupled from orders, making it easy to define custom payment methods with
+their own processing logic.
+
+Payment methods typically represent a payment processor. Payment methods will
+process credit card payments, and may also include non credit card payment
+methods such as Check or StoreCredit, which are provided in Solidus by default.
+
+The `Payment` model in Solidus tracks payments against [Orders](orders).
```

Comment:
Right now the `[Orders](orders)` link here doesn't go anywhere, and we don't have an equivalent page worth linking to yet.

Reply:
I'll remove that link